src/authInfo.ts

```diff
@@ -98,6 +98,13 @@ export interface AccessTokenOptions {
   instanceUrl?: string;
 }
 
+/* eslint-disable camelcase */
+type UserInfo = AnyJson & {
```

Comment:
These are the relevant attributes obtained by hitting the current user API endpoint.